GIBSDownloader/coordinate_utils.py

```diff
@@ -31,22 +55,33 @@ def from_str(cls, input_str):
 
         return Rectangle(bl_coords, tr_coords)
 
-    # Calculates the necessary width and height of image encompassing Rectangle's bounding box
-    # Taken from https://github.com/NASA-IMPACT/data_share
-    def calculate_width_height(self, resolution: float):
-        """
-        resolution: represents the pixel resolution, i.e. km/pixel. Should be a value from this list: [0.03, 0.06, 0.125, 0.25, 0.5, 1, 5, 10]
+    def calculate_width_height(self, resolution):
+        """ 
+        Calculates the necessary width and height of image encompassing Rectangle's bounding box
+
+        Parameters:
+            resolution (float): represents the pixel resolution, i.e. km/pixel. 
+                resolution should be a value from this list: 
+                    [0.03, 0.06, 0.125, 0.25, 0.5, 1, 5, 10]
+
+        Returns:
+            (width, height) of Rectangle object given `resolution`
+        
+        Code taken from: https://github.com/NASA-IMPACT/data_share
         """
         KM_PER_DEG_AT_EQ = 111.
         km_per_deg_at_lat = KM_PER_DEG_AT_EQ * np.cos(np.pi * np.mean([self.bl_coords.y, self.tr_coords.y]) / 180.)
```

Comment:
I fully trust that you are using the correct formula here. Might be nice to have context on where you got the formula from, such as a link to a section on Wikipedia or a mention of a reference book with page number that someone in the future can look up.

Reply:
The formula is taken from NASA IMPACT's example code, but I have added some further explanation/links to why the constant 111 is used, and how the other terms are derived.